pkg/minikube/cluster/cluster_test.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright (C) 2016 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cluster
+
+import (
+	"testing"
+	"github.com/stretchr/testify/assert"
+	"github.com/minishift/minishift/pkg/minikube/constants"
+	"path/filepath"
+	"fmt"
```

Comment:
@hferentschik Ah that got left when I noticed few Println statements and removed. Will update it